### PR TITLE
OpenAI Vector Rally Track

### PR DESCRIPTION
**Pull request by Mikep86**
Add a rally track for OpenAI vector benchmarking. Based on the `cohere_vector` track.

SEARCH-84

### CODE REVIEW

File: openai_vector/README.md
Comment (Mikep86) [inline openai_vector/README.md:1]: The README is incomplete, we need to decide on the public bucket to upload the raw dataset files to so I can document how to generate the document dataset and complete it.

File: openai_vector/queries.json.bz2
Comment (Mikep86) [inline openai_vector/queries.json.bz2:1]: I compressed the queries file because the uncompressed version was over 100 MB and couldn't be added to git

**Review by afoucret — APPROVED** [review]
Nice job!!!

File: openai_vector/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+## OpenAI vector track
+
+This track benchmarks the [NQ dataset](https://huggingface.co/datasets/BeIR/nq) enriched with embeddings generated using OpenAI's [`text-embedding-ada-002` model](https://openai.com/blog/new-and-improved-embedding-model).
+
+### Generating the document dataset
+
+To rebuild the document dataset:
+
+1. Install Python dependencies listed in `_tools/requimements.txt`
```
Comment (kderusso) [inline openai_vector/README.md:9]:
```suggestion
1. Install Python dependencies listed in `_tools/requirements.txt`
```

File: openai_vector/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "text": "In accounting, minority interest (or non-controlling interest) is the portion of a subsidiary corporation's stock that is not owned by the parent corporation. The magnitude of the minority interest in the subsidiary company is generally less than 50% of outstanding shares, or the corporation would generally cease to be a subsidiary of the parent.[1]",
+  "emb": [-0.01128644309937954, -0.02616020105779171, 0.012801663018763065, ...]
+}
+```
+
+### Generating the queries
+
+To rebuild the `queries.json.bz2` file:
+
+1. Install Python dependencies listed in `_tools/requimements.txt`
```
Comment (kderusso) [inline openai_vector/README.md:30]:
```suggestion
1. Install Python dependencies listed in `_tools/requirements.txt`
```

**Review by kderusso — APPROVED** [review]
Nice work!

**Pull request merged**